### ISSUE TITLE
Burp 2 on Windows won't allow NULL ciphers?

### ISSUE THREAD

**Issue by Sjoerd-** [grke/burp#530]
Since I use Burp either on my home network or through SSH tunnels I have no need for its encryption. I've been using Burp with ssl_ciphers=NULL for a long time but can't get that to work on an updated (2.1.0) Windows client. Should that still be possible?

**Comment by grke** [comment]
The difference between 2.0.54 and 2.1.0 Windows client is that 2.1.0 now uses openssl 1.1.0.
Does ssl_ciphers=NULL work on 2.0.54?

**Comment by Sjoerd-** [comment]
Yes, now that I had the chance to test on that client I can confirm that NULL ciphers work on 2.0.54.

**Comment by grke** [comment]
Hmm. I think maybe this requires that I enable 'weak ciphers' when compiling openssl for Windows. I will try it at some point.

**Comment by grke** [comment]
My burp server has openssl-1.0.1t.
The Windows burp client has openssl-1.1.0c.

I tried 'weak ciphers', and it didn't work.
I tried enable ssl3, and it didn't work.

I will try running a burp server with openssl-1.1.0c when I can, and see what happens.

I guess it means that the openssl people have disabled some things in their 1.1.0 versions, but I cannot
find anything that explains exactly what is happening here.
